### PR TITLE
{AKS} `az aks create/update`: Adopt new sku naming

### PR DESCRIPTION
**Pull request by FumingZhang**
**Related command**

- `az aks create`
- `az aks update`

**Description**

Adopt new sku naming since API version 2023-02-01 (see [rest-api-specs](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/containerservice/resource-manager/Microsoft.ContainerService/aks/stable/2023-02-01/managedClusters.json#L4863)).
- Change sku.name from `Basic` to `Base`.
- Change the value `Paid` in sku.tier to `Standard`.

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
AKS

**Review by xiangwei36 — APPROVED** [review]
LGTM

File: src/azure-cli/azure/cli/command_modules/acs/managed_cluster_decorator.py [hunk 1]
```diff
@@ -4023,7 +4023,7 @@ def _get_uptime_sla(self, enable_validation: bool = False) -> bool:
                 self.mc.sku and
                 self.mc.sku.tier is not None
             ):
-                uptime_sla = self.mc.sku.tier == "Paid"
+                uptime_sla = self.mc.sku.tier == "Standard"
```
Comment (zhoxing-ms) [inline src/azure-cli/azure/cli/command_modules/acs/managed_cluster_decorator.py:4026]: I'd like to confirm that whether these value change for `tier` will cause a breaking change to customers?

Comment (xiangwei36) [reply src/azure-cli/azure/cli/command_modules/acs/managed_cluster_decorator.py:4026]: The breaking change happens on the rp side api v2023-02. This is documented [here](https://learn.microsoft.com/en-us/azure/aks/free-standard-pricing-tiers). Once CLI is updated, the doc will be updated accordingly.

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**